doc/api/child_process.md

```diff
@@ -147,7 +147,9 @@ added: v0.1.90
 * Returns: {ChildProcess}
 
 Spawns a shell then executes the `command` within that shell, buffering any
-generated output.
+generated output. When specifying paths, spaces need to be dealt with 
```

Comment:
@sam-github I'll send another commit to update this PR to include these changes as well as those proposed by @Trott so that everyone can see a clean start point.